tests/test_sycl_queue.py

```diff
@@ -1232,3 +1232,16 @@ def test_take(device):
     result_queue = result.get_array().sycl_queue
 
     assert_sycl_queue_equal(result_queue, expected_queue)
+
+
+@pytest.mark.parametrize(
+    "device",
+    valid_devices,
+    ids=[device.filter_string for device in valid_devices],
+)
+def test_indices(device):
```

Comment:
need to check with both values of `sparse`, since different creation functions are used inside `indices` (here and in `tests/test_usm_type.py`)